packages/analytics/analytics-chart/src/index.ts

```diff
@@ -2,10 +2,14 @@ import AnalyticsChart from './components/AnalyticsChart.vue'
 import SimpleChart from './components/SimpleChart.vue'
 import TopNTable from './components/TopNTable.vue'
 import CsvExportModal from './components/CsvExportModal.vue'
+import useChartSelectedRange from './composables'
 
 export { AnalyticsChart, SimpleChart, TopNTable, CsvExportModal }
 
 export * from './types'
 export * from './enums'
 export * from './utils/colors'
 export * from './utils/customColors'
+export * from './utils/constants'
```

Comment:
@filipgutica you prefer that I move `formatTime` to `analytics-utilities` instead?

https://github.com/Kong/public-ui-components/blob/main/packages/analytics/analytics-utilities/src/format.ts